### ISSUE TITLE
Fail to ingesting metadata from prestosql(340)

### ISSUE THREAD

**Issue by njalan** [datahub-project/datahub#4512]
I am trying to import prestosql(340) metadata and faced error message **TrinoUserError: TrinoUserError(type=USER_ERROR, name=MISSING_CATALOG_NAME, message="line 2:6: Catalog must be specified when session catalog is not set",** Below is my config:
source:
  type: sqlalchemy
  config:
    platform: trino
    connect_uri: trino://xxx:xxx@xxxxx:8443/hive/xxx
    options:
      connect_args:
          catalog: hive
          schema: xxxx
          verify: False
    domain:
          "urn:li:domain:xxxx":
            allow:
              - ".*"
sink:
    type: datahub-rest
    config:
        server: 'http://xxxxx:8080/' (edited)

is there anything wrong with my configuration? 
If trino driver is not working is there any other solution for prestosql (340 is the version)?

**Comment by maggiehays** [comment]
@mayurinehate can you take a look a this one, please?

**Comment by mayurinehate** [comment]
@njalan Have you tried setting include_views to False ? It may work with it if the error is due to [this issue](https://github.com/trinodb/trino/issues/6294) in presto.

By the way, which sqlalchemy dialect pypi package are you using ? I doubt if any trino dialect supports connecting to prestosql(340). I think you will need to use sqlalchemy source with [presto sqlalchemy dialect](https://github.com/dropbox/PyHive#sqlalchemy).

**Comment by njalan** [comment]
@mayurinehate  My purpose is to import presto views so I can't set  include_views to False. How do I use sqlalchemy source with [presto sqlalchemy dialect](https://github.com/dropbox/PyHive#sqlalchemy)?  Set platform as Presto?

**Comment by mayurinehate** [comment]
yes. You can keep `platform` same if you want. Just change the connect_uri to use `presto://` instead of `trino://`

**Comment by njalan** [comment]
@mayurinehate  I am able to ingest presto view now. But faced some data type issue:  Did not recognize type 'varchar(2)' of column 'model'   Did not recognize type 'timestamp(3) with time zone' of column 'mms_time'

**Comment by mayurinehate** [comment]
Hey @njalan , it seems that pyhive presto dialect does not recognize these data types. Unfortunately, I don't seem to find any easy solution to this that does not involve active development. This issue would be solved if presto dialect starts supporting these types.

**Comment by njalan** [comment]
@mayurinehate I think we can close this issue.

**Comment by mayurinehate** [comment]
yes, please go ahead. cc: @maggiehays
